server/lib/config/index.js

```diff
@@ -97,13 +133,24 @@ class Config {
       if (
         key.startsWith('SQLPAD_') &&
         !isConnectionEnv(key) &&
-        !key.startsWith('SQLPAD_SERVICE_')
+        !key.startsWith('SQLPAD_SERVICE_') &&
```

Comment:
The exception No longer needed I think

Reply:
I was wondering about that... wasn't sure if its going to be a common message if it should be suppressed? But then its maybe inconsistent. Will remove